platform/windows/display_server_windows.cpp

```diff
@@ -4658,7 +4658,11 @@ DisplayServerWindows::DisplayServerWindows(const String &p_rendering_driver, Win
 			SetProcessDpiAwareness_t SetProcessDpiAwareness = (SetProcessDpiAwareness_t)GetProcAddress(Shcore, "SetProcessDpiAwareness");
 
 			if (SetProcessDpiAwareness) {
-				SetProcessDpiAwareness(SHC_PROCESS_SYSTEM_DPI_AWARE);
+				if ((os_ver.dwMajorVersion > 6) || ((os_ver.dwMajorVersion == 6) && (os_ver.dwMajorVersion >= 3))) {
```

Comment:
The version check should be unnecessary. `SetProcessDpiAwareness` is only available since Windows 8.1, so this path will practically always be taken.